components/search-page/SearchCollectiveCard.js

```diff
@@ -92,7 +92,7 @@ const SearchCollectiveCard = ({ collective, ...props }) => {
           )}
           {collective.description && (
             <Container fontSize="12px">
-              <Flex alignItems="center" justifyContent="space-between" mt={21.5} mb={4.5}>
+              <Flex alignItems="center" justifyContent="space-between" mt={10} mb={4.5}>
```

Comment:
We don't want to make changes to the general design of the card. The solution should be focused on reducing the size of the tags.